### DISCUSSION TITLE
Open bracket surround without space

### DISCUSSION THREAD

**Discussion by genepg** [kylechui/nvim-surround#236] [Polls]
I hope to switch the default for adding a surround bracket without space in the command of opening bracket `(`, `{`, `[`, `<`, and the closing bracket add space.

I know there is a way to config this. However, I think using the opening bracket should be more common because most developers usually use auto pairs and only press the opening bracket.

**Comment by kylechui** [comment]
IMO the decision is more or less arbitrary, and the design choice was made to be in line with tpope's `vim-surround`.

**Comment by gaoqiangks** [comment]
It's a pleasure to see this feature, since I seldom press the closing bracket.

**Comment by gaoqiangks** [comment]
BTW  how to config this?

**Reply by kylechui** [reply]
See `:h nvim-surround.config.surrounds`.

**Reply by gaoqiangks** [reply]
> See `:h nvim-surround.config.surrounds`.

Thank you very much!

**Comment by mukul-mpac** [comment]
I second that, opening brackets should not put a space while the closing bracket can!
